ext/session/session.c

```diff
@@ -2470,13 +2470,15 @@ PHP_FUNCTION(session_start)
 							zend_string *tmp_val;
 							zend_string *val = zval_get_tmp_string(value, &tmp_val);
 							if (php_session_start_set_ini(str_idx, val) == FAILURE) {
-								php_error_docref(NULL, E_WARNING, "Setting option '%s' failed", ZSTR_VAL(str_idx));
+								php_error_docref(NULL, E_WARNING, "Option \"%s\" cannot be set", ZSTR_VAL(str_idx));
```

Comment:
"failed" seems better than "cannot be set". The latter implies that it cannot be set at all, not that the specific value wasn't accepted.

Reply:
Agreed! Will revert this one